misk/src/main/kotlin/misk/tracing/backends/zipkin/ZipkinReporterConfig.kt

```diff
@@ -0,0 +1,21 @@
+package misk.tracing.backends.zipkin
+
+import misk.config.Config
+
+data class ZipkinReporterConfig(
+  /**
+   * Maximum backlog of span bytes reported vs sent. Corresponds to ReporterMetrics.updateQueuedBytes.
+   * Default 1% of heap
+   */
+  val queued_max_bytes: Int?,
```

Comment:
any way to inline these defaults?

Reply:
So, we had the same debate with Jaeger. Both of these tracers don't expose their defaults. Is it worth just copying them over anyway?

Reply:
no. was just asking